### PR TITLE
10 missing on air functionality

### LINKED ISSUE #10: Missing "On Air" functionality
Consideration of "On Air" functionality. 

It looks like the **ATEMstd** library has a public method that can return "On Air" status (getKeyerOnAirEnabled). My hope is this will work as intended, otherwise I will have to replicate the changes AronHetLam made to his version of the ATEMmin library. My preference is to use an existing method where possible. 

The reason I cannot just substitute libraries is that the ATEMmin does not support some of the calls Im making.

### PR DESCRIPTION
**Pull request by helo-head**
Should be ready for review

### CODE REVIEW

File: Tallly-Core2-V11.ino [hunk 1]
```diff
@@ -48,14 +49,17 @@ boolean writeEEconfig();
...
+// ATEM Connection, On Air and Battery Status Locations
 #define ACS_XLOC 120  // 312 Right Corner : 120 Second Column
 #define ACS_YLOC 7    // 7 First Row
 #define ACS_SIZE 6    // 7 Standard Size:
 #define BS_VYLOC 0    // 110 Second Row : 0 First Row
 #define BS_VXLOC 170  // 210 Left of Battery : 170 Third Column
 #define BS_BYLOC 0    // 109 Second Row : 0 First Row
 #define BS_BXLOC 265  // 285 Far Right : 265 Forth Column
+#define OAS_XLOC 135  // Right of center second row
+#define OAS_YLOC 0    // First Row
```
Comment (brianteeman) [inline Tallly-Core2-V11.ino]:
```suggestion
#define OAS_YLOC 7    // First Row
```

File: Tallly-Core2-V11.ino [hunk 2]
```diff
@@ -48,14 +49,17 @@ boolean writeEEconfig();
...
+#define OAS_SIZE 7    // Standard Size
```
Comment (brianteeman) [inline Tallly-Core2-V11.ino]:
```suggestion
#define OAS_SIZE 6    // Standard Size
```

File: Tallly-Core2-V11.ino [hunk 3]
```diff
@@ -1151,7 +1162,21 @@ void loop() {
...
+    atemOAStatus = AtemSwitcher.getStreamStreaming();
+    if (atemOAStatus != atemLastOAStatus) {
+      atemLastOAStatus = atemOAStatus;
+      M5.Lcd.fillCircle(ACS_XLOC, ACS_YLOC, ACS_SIZE, atemOAStatus ? TFT_RED : TFT_LIGHTGREY);  // ATEM "On Air" Status
+    }
+  } else {
+    M5.Lcd.fillCircle(OAS_XLOC, OAS_YLOC, OAS_SIZE, TFT_LIGHTGREY);    // If ATEM not connected then "On Air" is unknown at best
+    atemOAStatus = false;
+    atemLastOAStatus = false;
+    }  // End "Aon Air" status check
```
Comment (brianteeman) [inline Tallly-Core2-V11.ino]:
```suggestion
    }  // End "On Air" status check
```

**Comment by brianteeman** [comment]
Done a quick review to test compiling and default status
I assume that it is supposed to be a full circle and not a half circle so changed the LOC values

Will test the actual onair status is working in a few hours when I have a bit more time.

File: Tallly-Core2-V11.ino [hunk 4]
```diff
@@ -1151,7 +1162,21 @@ void loop() {
...
-  }  // End connection status loop
+  }  // End connection status
+  
+  
+  // Update ATEM "On Air" status
+  if (atemConStatus) {
+    atemOAStatus = AtemSwitcher.getStreamStreaming();
+    if (atemOAStatus != atemLastOAStatus) {
+      atemLastOAStatus = atemOAStatus;
+      M5.Lcd.fillCircle(ACS_XLOC, ACS_YLOC, ACS_SIZE, atemOAStatus ? TFT_RED : TFT_LIGHTGREY);  // ATEM "On Air" Status
```
Comment (brianteeman) [inline Tallly-Core2-V11.ino]:
```suggestion
      M5.Lcd.fillCircle(OAS_XLOC, OAS_YLOC, OAS_SIZE, atemOAStatus ? TFT_RED : TFT_LIGHTGREY);  // ATEM "On Air" Status
```

I think you made a copy paste error here. as when you are on air you are changing the color of the connection icon.

**Comment by helo-head** [comment]
Assuming the streaming/OnAir code works I need to add one more small bit of code to cover the case where someone wants to run this on a mini that does not support streaming.  Based on my current track record you will probably need to test it to make sure I don't break anything :)

**Comment by brianteeman** [comment]
> Assuming the streaming/OnAir code works

sorry forgot to say that it worked

**Comment by brianteeman** [comment]
am I correct that the condition is not dynamic but by manually changing the value of #define OAS_SUPP true 

If that is correct then the code looks ok and you just need to mention how to disable the ON AIR in the readme

**Comment by helo-head** [comment]
Added conditional OAS support, updated readme documentation to reflect Mini Pro+ requirement for OAS. 

Im debating the wisdom of changing filename and code in the same commit as you loose visibility into the code change. Not even sure a filename change is needed but originally did it to provide clarity.

**Comment by helo-head** [comment]
Hopefully its clear how to enable ... Im going to leave it enabled by default since the Mini (no pro) is no longer being sold.

**Pull request merged**